packaging/etc/engine-config/engine-config.properties

```diff
@@ -73,6 +73,10 @@ MaxNumOfVmSockets.description="Max Number of Socket per Virtual Machine; depreca
 MaxNumOfVmSockets.type=Integer
 MaxNumOfCpusCoefficient.description="The factor between max VM CPUs and configured VM CPUs; effective since compatibility level 4.6"
 MaxNumOfCpusCoefficient.type=Integer
+ManyVmCpus.description="How many vCPUs are considered to be many in maximum vCPU number calculations"
```

Comment:
not sure how easy it will be for users to understand what this means, but I don't have a better suggestion and we don't expect users to tune it until they face an issue, and in that case we can refer them to this config value (but it is worth mentioning it in the deep dive)